src/runtime/pkg/direct-volume/utils.go

```diff
@@ -0,0 +1,95 @@
+package volume
```

Comment:
In this design, the CSI plugin will interact with kata-runtime CLI online, but will still likely vendor this package for the MountInfo structure, correct?

It isn't expected that the CSI plugin will need or call these public functions?

Reply:
That's correct. CSI plugin shouldn't need to call these public functions. It does need to access the MountInfo structure.